### ISSUE TITLE
Please make a release for distributions. r479 is old.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mp4v2/mp4v2#133]
```
Please make a 2.0.0 release, even a 2.0.0-beta if you like. I've been in 
contact with various distribution maintainers (ArchLinux, Debian), and most of 
them are reclutant in adding r479 because it's

a) snapshot
b) badly out of date

Releasing 2.0.0 or a beta of it would encourage distribution maintainers to 
upgrade.

I've called r479 version 1.9.1_p479 myself and added it to Gentoo and ported 
all the reverse dependencies (consumers) to it's API, but I'm now receiving bad 
feedback from users with style of "It's a *curseword* snapshot, and my 
favourite pet project doesn't build with it"

So please please, make a release, even a alpha or a beta
```

Original issue reported on code.google.com by `ssuomine...@gmail.com` on 19 May 2012 at 5:33

**Comment by GoogleCodeExporter** [comment]
```
I agree...

I haven't paid enough attention to this project lately (sorry, not exactly a 
ton of spare time).  But I think current trunk is stable enough to release 
with, so I'll attempt to get that out the door this week.


```

Original comment by `jnor...@logitech.com` on 19 May 2012 at 7:46

**Comment by GoogleCodeExporter** [comment]
```
Okay, I've released an official 2.0.0 version of the library.  

It's worth noting there have been breaking API changes between 2.0.0 and 1.9.1, 
which is why we've incremented a major version--most notably, a metadata API 
that has been deprecated since 1.9.0 has been officially removed, along with a 
few minor changes.  Note that most of these breaking changes were made before I 
assumed ownership of this project, so please don't shoot the messenger.

So regarding "favorite pet project" blah doesn't build with it....I would 
sincerely like to know why so I can fix it as best I can.  So please direct 
those users my way if you can.
```

Original comment by `kid...@gmail.com` on 21 May 2012 at 4:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Seriously, thanks, if I were rich I'd donate something... ;-)  If you happen to 
be around Finland, I'll offer you some free beer at least. Heh.

Everything I had patched for r479 still compiled against 2.0.0, even faac. 
Seems to work too. Great.

Thanks again.
```

Original comment by `ssuomine...@gmail.com` on 21 May 2012 at 3:04
